enterprise/frontend/src/embedding-sdk/index.ts

```diff
@@ -11,6 +13,10 @@ import "sdk-ee-plugins";
 // Imports which are only applicable to the embedding sdk, and not the new iframe embedding.
 import "sdk-specific-imports";
 
+// Enable SDK mode as we are in the SDK bundle
+// This applies to SDK derivatives such as new iframe embedding.
+EMBEDDING_SDK_CONFIG.isSdk = true;
```

Comment:
We import the `"embedding-sdk"` from `SdkIframeEmbedRoute.tsx` for new iframe embedding too, so this flag gets set both for the SDK and new iframe embed.